.circleci/config.yml

```diff
@@ -31,6 +31,7 @@ jobs:
       - checkout
       - run:
           name: Run tests and code coverage
+          no_output_timeout: 800s
```

Comment:
This was leftover from when I was experimenting with the tests trying to resolve the hanging, but shouldn't be needed anymore, so I'll remove it.

The thing is that `go test` has the same timeout, which wouldn't be reached if CircleCI times out after the default 10m of no output. But since that should be from the last time there was any output, the default is probably fine. So yeah, will remove.

Reply:
Removed in latest push.